.github/workflows/run-archiver.yml

```diff
@@ -68,7 +68,13 @@ jobs:
           # ZENODO_TOKEN_UPLOAD: ${{ secrets.ZENODO_TOKEN_UPLOAD }}
           # ZENODO_TOKEN_PUBLISH: ${{ secrets.ZENODO_TOKEN_PUBLISH }}
         run: |
-          conda run -n pudl-cataloger pudl_archiver --sandbox --datasets ${{ matrix.dataset }}
+          conda run -n pudl-cataloger pudl_archiver --sandbox --datasets ${{ matrix.dataset }} --sumary-file ${{ matrix.dataset }}_run_summary.json
+
+      - name: Archive code coverage results
+        uses: actions/upload-artifact@v3
+        with:
+          name: $${{ matrix.dataset }}-run-summary
+          path: ${{ matrix.dataset }}_run_summary.json
```

Comment:
For now, we'd only access these through the Actions UI, right?

Reply:
Yeah right now that's the case. I've thought about adding a json file to the actual depositions, so there will be a history right in the deposition. Do you think this would be worth doing?

Reply:
Sometime down the line, I guess that could be useful. But this seems fine for now.